documentation/internal/Hardening.md

```diff
@@ -486,3 +487,52 @@ volumes:
 ...
 ```
 After this, restart the pod to reflect the changes and verify that the secret is mounted to the pod at the specified mount point.
+
+## Use strong cryptographic ciphers for API server
+
+**Kube-bench Identifier**:
+
+* 1.2.31
+
+### Strong Cyrptographic Ciphers suggested by CIS
+* TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+* TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+* TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305
+* TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+* TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305
+* TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+
+### Mnaual Application for Strong Cyrptographic Ciphers for API server on pr-installed cluster
+
+Edit the API server pod specification `file /etc/kubernetes/manifests/kube-apiserver.yaml` on the control all plane nodes and add below parameter to the API server arguments
```

Comment:
Maybe it needs to be preserved these changes in `kubeadm-config` configMap for further maintenance procedures?